src/main/resources/static/templates/docs.html

```diff
@@ -706,6 +706,97 @@ <h1>Reactor RabbitMQ</h1>
                 <a href="/docs/rabbitmq/snapshot/reference/">Snapshot</a>
             </div>
         </article>
+        <article>
+            <div class="title">
+                <h1>BlockHound</h1>
+                <p>Java agent to detect blocking calls from non-blocking threads</p>
+                <div class="version">
+                    <table>
+                        <tr><td>Version</td>
+                            <td><a
+                                    href="http://github.com/reactor/BlockHound/releases"
+                                   title="See all BlockHound releases on GitHub"
+                                   class="fa fa-archive">&nbsp;</a>
+                            </td>
+                            <td>Release Train</td>
+                        </tr>
+                        <!--RELEASEs from in-progress release train first-->
+                        <tr th:if="${upcoming.blockhoundVersion} != null">
```

Comment:
all these conditionals depend on adding a `blockhoundVersion` to the `Bom.java`... If blockhound is likely to be released out of the reactor release train, then you'll need to remove template variables and go back to manually editing/commenting out sections that do or don't make sense (milestones vs release, most likely)